iquip/apps/builder.py

```diff
@@ -664,7 +664,7 @@ def __init__(
     def scanArguments(self) -> Dict[str, Any]:
         """Overridden."""
         sequenceText = self.sequenceEdit.text()
-        sequence = [float(x) for x in sequenceText.split()]
+        sequence = [float(x) * self.scale for x in sequenceText.split()]
```

Comment:
As I found in [ARTIQ](https://github.com/m-labs/artiq/blob/56bd975a347155181cf1fdd5fc383742d22daf57/artiq/gui/entries.py#L532), in `ExplicitScan`, the scale argument may be not used.
Could you check it?

Reply:
Oh, you are right! Explicit scan does not use any unit so it should not use the scale parameter.